OneCommunityGlobal/HGNRest
# Abi Unit and Integration tests for getActionItem routes and Controllers

# Description
Implementing unit and integration tests for the `actionItemController` and `actionItemRoutes`. These tests cover the post and get methods for the routes/controllers

## Related PRS (if any):
N/A
…

## Main changes explained:
- Implements unit tests for the getactionItem and postactionItem method in the getActionController.
- Implements integration tests for the get and post routes in `actionItemRouter`
- Tests the positive and negative and applicable edge case for all test cases
…

## How to test:
1. check into current branch
2. do `npm install` 
3. run `npm run test actionItem`
4. verify that all tests pass.

## Screenshots or videos of changes:

![Screenshot 2024-03-27 at 11 43 11 AM](https://github.com/OneCommunityGlobal/HGNRest/assets/59585815/f7c9e00b-ec8a-4a73-8d9c-53b77114ec77)
